clickhouse/columns/numeric.cpp

```diff
@@ -10,7 +10,14 @@ ColumnVector<T>::ColumnVector()
 }
 
 template <typename T>
-ColumnVector<T>::ColumnVector(const std::vector<T>& data)
+ColumnVector<T>::ColumnVector(const std::vector<T> & data)
+    : Column(Type::CreateSimple<T>())
+    , data_(data)
+{
+}
+
+template <typename T>
+ColumnVector<T>::ColumnVector(std::vector<T> && data)
     : Column(Type::CreateSimple<T>())
     , data_(data)
```

Comment:
Then, this one should be passed further as an rvalue here: `, data_(std::move(data))`